### DISCUSSION TITLE
Broken song duration visualizer and notes coulour on android

### DISCUSSION THREAD

**Discussion by LJW21-GitHub** [ppy/osu#28077] [Q&A] [answered]
### Type

Cosmetic

### Bug description

When trying to skin my Android Osu!, I've noticed that the song duration visualizer was broken and was still the Argon skin one.
Also, the notes were looking flat, and in one of the skins I tried the notes were supposed to be standardized, it wasn't.

### Screenshots or videos

![Screenshot_20240503-064709](https://github.com/ppy/osu/assets/142221566/a55715d4-0cc1-4b59-bdb8-9a3bf471b351)

### Version

2024.412.1

### Logs

[1714322331.auth.log](https://github.com/ppy/osu/files/15196413/1714322331.auth.log)
[1714322331.database.log](https://github.com/ppy/osu/files/15196414/1714322331.database.log)
[1714322331.input.log](https://github.com/ppy/osu/files/15196415/1714322331.input.log)
[1714322331.network.log](https://github.com/ppy/osu/files/15196416/1714322331.network.log)
[1714322331.performance.log](https://github.com/ppy/osu/files/15196417/1714322331.performance.log)
[1714322331.runtime.log](https://github.com/ppy/osu/files/15196418/1714322331.runtime.log)

**Comment by bdach** [comment]
> I've noticed that the song duration visualizer was broken and was still the Argon skin one

I don't get what this means, and I don't see any "argon song duration visualiser" on the attached screenshot

> the notes were looking flat

I don't get what this means

> in one of the skins I tried the notes were supposed to be standardized

I don't get what this means

Please accurately describe your problem. With screenshots showing what the problem is, precisely. Maybe also attach the skin too.

**Reply by LJW21-GitHub** [reply]
On the top right, it's the default legacy song duration visualizer, and it should looks like the one in the demo video

**Reply by bdach** [reply]
What "demo video"?

If you use a legacy skin you will see the legacy song progress. This is intended. If you want the argon one, you can add it via the skin editor.

**Reply by LJW21-GitHub** [reply]
When I say colours are normalized I mean that all the notes were supposed to be orange or blue, and some of they appears red, purple, or any default combo colours

**Reply by bdach** [reply]
> When I say colours are normalized I mean that all the notes were supposed to be orange or blue

can't comment on this any more without seeing what the actual skin is, and maybe even what beatmap this was on. could have been beatmap combo colours being active or something

**Reply by LJW21-GitHub** [reply]
(I'm using V3)

**Reply by LJW21-GitHub** [reply]
Here's the skin demo : https://www.youtube.com/watch?v=IxGIaomTvZg

**Reply by bdach** [reply]
> Well, demo video I was about to post when you closed the channel, but here it is ;

the song progress thing is broken because the skin you're trying to use was relying on bad skinning hacks which rely on specific screen resolution to work. we will not be fixing this.

**Reply by LJW21-GitHub** [reply]
Oh, ok... But how can I fix this ?

**Reply by LJW21-GitHub** [reply]
And I've tried a second skin, the same bug occurred : https://www.reddit.com/r/OsuSkins/s/KuHc9HdYaH

**Reply by LJW21-GitHub** [reply]
I think I understand what you mean with the specific screen resolution but it's not an issue, Lazer skin edit can easily fix that, the problem is the duration visualizer and notes apparence herself

**Reply by Joehuu** [reply, marked as answer]
@LJW21-GitHub You probably have beatmap colours on.  As for the WhiteCat skin, you probably downloaded the wrong one out of the many different variations in the messy downloads page. The demo didn't feature any blue slider border skin.
